pkg/util/github/github.go

```diff
@@ -125,7 +125,7 @@ func DownloadOpenShiftReleaseBinary(binaryType OpenShiftBinaryType, osType minis
 		return errors.Wrap(err, fmt.Sprintf("Cannot download OpenShift release asset %d", assetID))
 	}
 	if len(url) > 0 {
-		glog.V(2).Infof("Downloading %s %s\n", binaryType.String(), *release.TagName)
+		fmt.Println(fmt.Sprintf("Downloading OpenShift binary '%s' in version '%s'", binaryType.String(), *release.TagName))
```

Comment:
Nitpicking. Do we need the the `in` in the sentence i.e. `Downloading OpenShift binary 'oc' in version 'v1.5.0-rc.0' ` VS `Downloading OpenShift binary 'oc' version 'v1.5.0-rc.0'`